src/hud_elements.cpp

```diff
@@ -922,6 +925,80 @@ void HudElements::custom_text_center(){
     }
 }
 
+void HudElements::image(){
+    const std::string& value = HUDElements.ordered_functions[HUDElements.place].value;
```

Comment:
`ifdef __linux__` the entire content of `image()` and `background_image()` is usually how we handle this so it doesn't fail in mingw